release/e2e.py

```diff
@@ -893,6 +894,9 @@ def wait_for_session_command_to_complete(create_session_command_result,
     start_wait = time.time()
     next_report = start_wait + REPORT_S
     while not completed:
+        result = sdk.get_session_command(session_command_id=scd_id)
+        completed = result.result.finished_at
+
```

Comment:
Same here, we will still sleep even if the loop is completed here

Reply:
yep